### DISCUSSION TITLE
npx sb init has no effet

### DISCUSSION THREAD

**Discussion by sandro768** [storybookjs/storybook#16660] [Help]
executing `npx sb init` doesn't do anything in a Typescript CRA project.

![image](https://user-images.githubusercontent.com/24352108/141131471-ea1cd075-4633-4f31-8565-d04c9cf2eadb.png)

Environment Info:

  System:
    OS: Linux 5.11 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (4) x64 Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
  Browsers:
    Chrome: 95.0.4638.69
    Firefox: 94.0

**Comment by shilman** [comment]
Can you try `npx sb@latest init` or `npx sb@next init`? i think there's something weird cached on your machine possibly
